src/lib/Preview.js

```diff
@@ -928,6 +928,10 @@ class Preview extends EventEmitter {
         // Whether annotations v4 buttons should be shown in toolbar
         this.options.showAnnotationsControls = !!options.showAnnotationsControls;
 
+        this.options.showAnnotationsHighlightText = !!options.showAnnotationsHighlightText;
```

Comment:
Was this removed earlier today in another PR?

Reply:
Yes it was, will remove, it got batched in with my rebase earlier.